www/js/survey/enketo/EnketoModal.tsx

```diff
@@ -63,6 +63,13 @@ const EnketoModal = ({ surveyName, onResponseSaved, opts, ...rest }: Props) => {
   useEffect(() => {
     if (!rest.visible || !appConfig) return;
     initSurvey();
+
+    // on dev builds, allow skipping survey with ESC
```

Comment:
Yes it will affect them. This change was just focused on making testing in emulator easier, which was only cumbersome with the onboarding survey.
The trip-level surveys have a "dismiss" button, while the first time through the onboarding survey does not (by design)

I don't know that the check for `__DEV__` was really needed, given that it is pretty hard to press `ESC` on a mobile app. But I did the principled thing and made sure it is only possible on dev builds